sdk/keyvault/keyvault-admin/README.md

```diff
@@ -34,16 +33,151 @@ TypeScript users need to have Node type definitions installed:
+Here's an example using the Azure CLI:
+
+```Powershell
+az storage account create --name MyStorageAccount --resource-group MyResourceGroup --location westus --sku Standard_LRS
+```
+
+#### Environment Variables
+
+With a Key Vault and a Storage account, you can fill the rest of the environment variables in the `.env` file used by [dotenv][dotenv]. You can look for more information about other required environment variables in our [sample.env](./sample.env) file.
+
+#### Create KeyVaultAccessControlClient
+
+Once you've populated the **AZURE_CLIENT_ID**, **AZURE_CLIENT_SECRET** and **AZURE_TENANT_ID** environment variables and replaced **your-vault-url** 
+with the above returned URI, you can create the [KeyVaultAccessControlClient][src-rbac-client]:
+
+```ts
+const credentials = new DefaultAzureCredential();
+const client = new KeyVaultAccessControlClient(vaultUrl, credentials);
+```
+
+#### Create KeyVaultBackupClient
 
-### Configuring your Key Vault
+Once you've populated the **AZURE_CLIENT_ID**, **AZURE_CLIENT_SECRET** and **AZURE_TENANT_ID** environment variables and replaced **your-vault-url** 
+with the above returned URI, you can create the [KeyVaultBackupClient][src-backup-client]:
+
+```ts
+const vaultUrl = `https://<MY KEY VAULT HERE>.vault.azure.net`;
+const credentials = new DefaultAzureCredential();
+const client = new KeyVaultBackupClient(vaultUrl, credentials);
+```
 
 ## Key concepts
 
-## Authenticating with Azure Active Directory
+### KeyVaultRoleDefinition
+
+A `KeyVaultRoleDefinition` is a collection of permissions. A role definition defines the operations that can be performed, such as read, write, 
+and delete. It can also define the operations that are excluded from allowed operations.
+
+KeyVaultRoleDefinitions can be listed and specified as part of a `KeyVaultRoleAssignment`.
+
+### KeyVaultRoleAssignment. 
+
+A `KeyVaultRoleAssignment` is the association of a KeyVaultRoleDefinition to a service principal. They can be created, listed, fetched individually, and deleted.
+
+### KeyVaultAccessControlClient
+
+A [KeyVaultAccessControlClient][src-rbac-client] provides both synchronous and asynchronous operations allowing for management of `KeyVaultRoleDefinition` and `KeyVaultRoleAssignment` objects.
+
+### KeyVaultBackupClient
+
+A [KeyVaultBackupClient][src-backup-client] provides both synchronous and asynchronous operations for performing full key backups, full key restores, and selective key restores.
+
+### Long Running Operations
+
+The Azure SDK for TypeScript and JavaScript offers a common abstraction for long running operations through the package [core-lro][corelro]. In particular, the operations done by the [KeyVaultBackupClient][src-backup-client] may take as much time as needed by the Azure resources, requiring a client layer to keep track, serialize and resume the operations through the lifecycle of the programs that wait for them to finish.
```

Comment:
it doesn't appear that `[corelro]` exists in the links below

Reply:
thank you!